packages/rocketchat-ui-sidenav/client/accountBox.coffee

```diff
@@ -11,12 +13,14 @@ Template.accountBox.helpers
 				visualStatus = t("invisible")
 		return {
 			name: Session.get('user_' + username + '_name')
-			status: Session.get('user_' + username + '_status')
+			status: if colorBlind then status + '-color-blind' else status
```

Comment:
I think you can set the status color variable to color blind variable instead of this, what you think?